packages/css-library/src/stylesheets/core.scss

```diff
@@ -36,6 +36,8 @@
 
 // ----- VA GENERAL (MAIN FILE/OTHER) ---- //
 @import 'base/va';
+@import 'base/fonts';
+@import 'base/utils';
```

Comment:
fonts is needed because it's the entry point for `uswds-fonts`, which contains all of the font-face settings for source sans pro web. Prior to this, even though we were using uswds-typography settings, it seems like our imports for fonts in vets-website were still relying on formation/core. 

utils is needed because it contains at least one style rule, `va-button-link` that is actively being used in vets-website, but we should definitely look to deprecate this moving forward.